heroku.cfg

```diff
@@ -0,0 +1,46 @@
+import os
+
+ENV = "prod"
+SECRET_KEY = os.environ['SECRET_KEY']
+
+DOORMAN_LOGGING_FILENAME = "sys.stdout"
+
+#SERVER_NAME = ''
+PREFERRED_URL_SCHEME = "https"
+
+# ensure we don't roll out production w/debug turned on
+DEBUG = False
+DEBUG_TB_ENABLED = False
+DEBUG_TB_INTERCEPT_REDIRECTS = False
+
+DOORMAN_ENROLL_SECRET = os.environ['ENROLL_SECRET']
+SQLALCHEMY_DATABASE_URI = os.environ['DATABASE_URL']
+BROKER_URL = os.environ['REDIS_URL']
+CELERY_RESULT_BACKEND = os.environ['REDIS_URL']
+
+DOORMAN_AUTH_METHOD = "google"
+DOORMAN_OAUTH_CLIENT_ID = os.environ['OAUTH_CLIENT_ID']
+DOORMAN_OAUTH_CLIENT_SECRET = os.environ['OAUTH_CLIENT_SECRET']
+DOORMAN_OAUTH_GOOGLE_ALLOWED_USERS = os.environ['OAUTH_ALLOWED_USERS']
+
+# mail config
+MAIL_SERVER = os.environ['MAIL_SERVER']
+MAIL_PORT = os.environ['MAIL_PORT']
+MAIL_USERNAME = os.environ['MAIL_USERNAME']
+MAIL_PASSWORD = os.environ['MAIL_PASSWORD']
+MAIL_DEFAULT_SENDER = os.environ['MAIL_DEFAULT_SENDER']
+MAIL_USE_SSL = True
+
+DOORMAN_ALERTER_PLUGINS = {
+    'debug': ('doorman.plugins.alerters.debug.DebugAlerter', {
+        'level': 'error',
+    }),
+
+    'email': ('doorman.plugins.alerters.emailer.EmailAlerter', {
+        'recipients': [
+            'test@example.com',
```

Comment:
Probably want to make this something like:

```'recipients': [email.strip() for email in os.environ['DOORMAN_MAIL_RECIPIENTS'].split(';')]```

so that you can configure a semicolon delimited list of users to email in the event an alert is triggered.